# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/base/nsIChannel.idl
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c netwerk/base/nsIChannel.idl
--- a/netwerk/base/nsIChannel.idl	Tue Feb 12 17:58:19 2019 +0100
+++ b/netwerk/base/nsIChannel.idl	Tue Feb 12 16:08:25 2019 +0000
@@ -156,11 +156,6 @@
     nsIInputStream open();
 
     /**
-     * Performs content security check and calls open()
-     */
-    nsIInputStream open2();
-
-    /**
      * Asynchronously open this channel.  Data is fed to the specified stream
      * listener as it becomes available.  The stream listener's methods are
      * called on the thread that calls asyncOpen and are not called until
@@ -188,15 +183,9 @@
      * channel is reopened.
      *
      * @param aListener the nsIStreamListener implementation
-     * @param aContext an opaque parameter forwarded to aListener's methods
      * @see nsIChannelEventSink for onChannelRedirect
      */
-    void asyncOpen(in nsIStreamListener aListener, in nsISupports aContext);
-
-    /**
-     * Performs content security check and calls asyncOpen().
-     */
-    void asyncOpen2(in nsIStreamListener aListener);
+    void asyncOpen(in nsIStreamListener aListener);
 
     /**************************************************************************
      * Channel specific load flags: